packages/compiler/.scripts/build-init-templates.ts

```diff
@@ -20,7 +20,7 @@ const builtInTemplates: Record<string, InitTemplate> = {
         label: "OpenAPI 3.1 document",
         options: {
           "emitter-output-dir": "{output-dir}/schema",
-          "openapi-versions": ["3.1.0"],
+          "openapi-versions": ["3.2.0"],
```

Comment:
you think it might not be better to wait a little before setting the default in templates to 3.2.0. Seeing how long it took for things to support 3.1.0, I worry this will have a not very interoperable output for a new user.